### PR TITLE
Check assignment

### PR DESCRIPTION
**Pull request by Dpalazzari**
This PR checks the assignment and makes sure it isn’t already true.

[#142745305]

### CODE REVIEW

File: app/models/assignment.rb [hunk 1]
```diff
@@ -10,4 +10,8 @@ def self.complete(describee_id, describer_id)
     assignment.update(completed?: true)
     assignment.save
   end
+
+  def self.valid?(describee_id, describer_id)
```
Comment (kheppenstall) [inline app/models/assignment.rb]: @Dpalazzari @wlffann Should this method be `invalid?`? It seems to return true if it is empty.

**Comment by kheppenstall** [comment]
👍 Thanks for doing this @Dpalazzari @wlffann It looks great. I just left one comment above.

File: app/models/assignment.rb (hunk 1 above)
Comment (Dpalazzari) [inline app/models/assignment.rb]: @kheppenstall I will make this change real quick.

**Comment by kheppenstall** [comment]
@Dpalazzari thanks for making that change! Merging in now.

**Pull request merged**